coder-login/main.tf

```diff
@@ -23,7 +23,7 @@ resource "coder_script" "coder-login" {
     CODER_DEPLOYMENT_URL : data.coder_workspace.me.access_url
   })
   display_name       = "Coder Login"
-  icon               = "/emojis/1f511.png"
+  icon               = "/favicons/favicon.svg"
```

Comment:
nice does this work?

Reply:
It turned out to be too dark. I have a link from svgur now.